README.md

```diff
@@ -6,11 +6,12 @@
 [![Open questions](https://img.shields.io/badge/Open-questions-pink.svg?style=flat-square)](https://github.com/theia-ide/theia/labels/question)
 [![Open bugs](https://img.shields.io/badge/Open-bugs-red.svg?style=flat-square)](https://github.com/theia-ide/theia/labels/bug)
 
-Theia is an extensible platform to develop full-fledged multi-language Cloud & Desktop IDE-like products with state-of-the-art web technologies.
+Theia is an extensible platform to develop full-fledged multi-language Cloud & Desktop IDE-like products with state-of-the-art web technologies. All languages which are implemented in the LSP [are supported](https://langserver.org/#implementations-server)
 
 ![Theia](https://user-images.githubusercontent.com/372735/33182625-0f6575f0-d075-11e7-8ec7-53801e3892bd.jpg)
 
 - [**Website**](#website)
+- [**Docker**](https://hub.docker.com/r/theiaide/theia/)
```

Comment:
It is already covered in `Getting Started` section. We could add subpoints in the TOC that one see them immediately.

Reply:
I have stopped to read after "for web developers" 

Why should it only be for web devs?

Reply:
Yeah, completely agree :D